packages/lumx-react/src/components/slider/Slider.tsx

```diff
@@ -265,6 +265,7 @@ const Slider: React.FC<SliderProps> = ({
                 handleBasicClasses({ prefix: CLASSNAME, theme, hasLabel: Boolean(label) }),
             )}
             onMouseDown={handleMouseDown}
+            aria-disabled={isDisabled}
```

Comment:
My screen reader doesn't seem to really use this info :man_shrugging: 
But it can't do any harm either I guess :smile: